Automattic/jetpack
# External Media: do not display buttons in caption field.

Fixes #37943

## Proposed changes:

Ensure the External Media options are not displayed when editing captions in image blocks.

### Other information:


## Jetpack product discussion

* No

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

> [!NOTE]
> It's worth testing this in both WordPress.com (where Gutenberg is active) and on a site without the Gutenberg plugin.

* Start with a site connected to WordPress.com.
* Go to Posts > Add New
* Add an Image block
    * You should be able to upload images from alternate sources like Pexels or Google Photos.
* Once you've uploaded the image, click to edit the caption
    * No External Media buttons should appear.
* In the post sidebar, click to add a Featured Image
    * The options to add images from an external source should be displayed.
* Try using the Gallery, Tiled Gallery, and Slideshow blocks too.
